src/Stratis.Bitcoin.Features.BlockStore.Tests/BlockStoreTests.cs

```diff
@@ -86,7 +86,9 @@ public BlockStoreTests()
 
             this.chainState = new ChainState();
 
-            this.blockStoreQueue = new BlockStoreQueue(this.chain, this.chainState, new StoreSettings(NodeSettings.Default(this.network)),
+            var blockStoreFlushCondition = new BlockStoreQueueFlushCondition(this.chainState);
```

Comment:
This class was introduced by @fassadlr a few commits ago, @sondreb I think you did not do the merge correctly

Reply:
Have done a new merge and fixes the conflicts, now the PR only contains the correct changes in 5 files.